### DISCUSSION TITLE
Double Enter and Nested Lists?

### DISCUSSION THREAD

**Discussion by Ivcota** [vslinko/obsidian-outliner#127] [Archive]
I want to start of saying that I love this plugin, and it's become essential for my Zettelkasten workflow. 

So I'm not sure if I'm doing something wrong, but whenever I double hit enter while the list is folded—like I'm trying to make a new sibling bullet—instead of it making a new sibling, it unfolds the list and makes 2 children bullets.

It only seems to work if I first unnest the list—then double hit enter.

I might not be aware that this is intended, but if not, I think it might be a bug. It happens with both of my Windows devices. 

Thanks a lot and I'm going to support as soon as my PayPal stops acting buggy and denying my attempts to donate 😭😂.

Thanks!

**Comment by vslinko** [comment]
@Ivcota Hi!

I'm not sure that I understand correctly. Could you please make screen recording?

**Reply by Ivcota** [reply]
Sure thing @vslinko :)

https://user-images.githubusercontent.com/18635074/117667231-8c5c5880-b159-11eb-9c3e-c160311e33a4.mp4

**Reply by Ivcota** [reply]
@vslinko Oh, it seems to be working in version 1.1.5 but not in the latest. Also, Paypal is working now lol—hopefully you can get a cup of coffee :)

**Reply by vslinko** [reply]
@Ivcota Now I got it. Thank you for your contribution!

Should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/1.1.9

**Reply by Ivcota** [reply]
> @Ivcota Now I got it. Thank you for your contribution!
> 
> Should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/1.1.9

Oh great, awesome! However, It seems to open up the nesting when you hit enter. I have the 1.1.5 installed on one device and hitting double enter does not unnest the list. Here's what 1.1.5 looks like: 

https://user-images.githubusercontent.com/18635074/118383165-66e3aa80-b5b0-11eb-8189-f15315d2c6bf.mp4

Can we get that feature back in a future update? Thanks!

**Reply by Ivcota** [reply]
Oh and here's an idea to consider. If you hit enter once on a nested list, it makes a sibling bullet. Like Just hit enter once and it replicates that result from the video above.

**Reply by vslinko** [reply]
Please start new discussions in ideas https://github.com/vslinko/obsidian-outliner/discussions/categories/ideas
